0000-catalog.md

```diff
@@ -0,0 +1,789 @@
+- Feature Name: Product Catalog
+- Start Date: 2019-07-12
+- RFC PR:
+- Hyperledger Grid Issue: [GRID-95](https://jira.hyperledger.org/browse/GRID-95)
+
+# Summary
+
+[summary]: #summary
+
+This RFC proposes a generic implementation for a Hyperledger Grid _Product
+Catalog_. Product Catalogs will be represented by a unique identifier, which
+will be referenced by the _catalog_products_ included in the catalog. That
+catalog_id can be used to share a grouping of products that can be shared with 
+one or more organizations. The Grid Catalog will contain 4 fields. A 
+`catalog_id`, a `catalog_owner`, a `name` for the catalog, and a repeated 
+field of PropertyValues for custom fields.
```

Comment:
Consider naming `PropertyValue` to `PropertyDefinition`

Reply:
`PropertyDefinition` is the term used to define a `PropertyValue` in the context of a Grid Schema. You can toss in `PropertyValues` without the need for a schema. 

Though if you wanted one or more Catalogs to be _required_ to have certain additional fields you'd need to write a Grid Schema that has `PropertyDefinitions` for the `PropertyValues` you wish to collect.